llvm/lib/Target/RISCV/RISCVAsmPrinter.cpp

```diff
@@ -626,7 +626,7 @@ void RISCVAsmPrinter::LowerHWASAN_CHECK_MEMACCESS(const MachineInstr &MI) {
     Sym = OutContext.getOrCreateSymbol(SymName);
   }
   auto Res = MCSymbolRefExpr::create(Sym, OutContext);
-  auto Expr = RISCVMCExpr::create(Res, RISCVMCExpr::VK_CALL, OutContext);
```

Comment:
Isn't this change implying that the only object file format we can target under RISCV is ELF? I would expect RISCVAsmPrinter to be format independent.